pkg/tfgen/generate_python.go

```diff
@@ -390,7 +390,7 @@ func (g *pythonGenerator) emitRawDocComment(w *tools.GenWriter, comment, prefix
 
 func (g *pythonGenerator) emitPlainOldType(w *tools.GenWriter, pot *plainOldType) {
 	// Produce a class definition with optional """ comment.
-	w.Writefmtln("class %s(object):", pyClassName(pot.name))
+	w.Writefmtln("class %s:", pyClassName(pot.name))
```

Comment:
This was a drive-by change, deriving from `object` explicitly isn't good Python style (Previously it was required in Python 2 for esoteric reasons, Python 3 considers it to be implicit).